### DISCUSSION TITLE
Different Ports for GUI and Client Registration?

### DISCUSSION THREAD

**Discussion by bloesss** [immense/Remotely#653] [General]
Hi,

we would like to make the GUI only reachable via VPN or Client Certificate. We can use a Nginx Reverse Proxy for this case.
But then we would need to use different ports for the Web GUI and the Client Registration.
Or is the _blazor directive used for client registration?

**Comment by Xevichr** [comment]
Would also like an answer to this question, but I assume they are very likely the same. On the container only 5000:5000 is used.
